modules/http4s/test/src/compliancetests/ProtocolComplianceTest.scala

```diff
@@ -22,11 +22,11 @@ import org.http4s._
 import org.http4s.client.Client
 import smithy4s.kinds.FunctorAlgebra
 import smithy4s.Service
-import smithy4s.example.test._
```

Comment:
idk about everyone else, but my github gave up trying to syntax highlight this file.